LICENSE.txt

```diff
@@ -28,3 +28,19 @@ INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
 CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
 THE POSSIBILITY OF SUCH DAMAGE.
+
+
+
+SciPy bundles a number of libraries that are compatibly licensed:
```

Comment:
I think it should also list the location of the bundled sources, as [debian copyright files](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/) do.

Reply:
Good idea. I like the Debian style better than the Apache one.

Reply:
The debian copyright file lists a number of other bundled items:
http://metadata.ftp-master.debian.org/changelogs/main/p/python-scipy/python-scipy_0.19.1-1_copyright

Reply:
Thanks, most of those make sense. I don't think ``ndimage`` does though (it's not a bundled lib but was written as part of SciPy AFAIK), nor does listing public domain files (not required, and there are a lot of them that aren't listed in the Debian file).